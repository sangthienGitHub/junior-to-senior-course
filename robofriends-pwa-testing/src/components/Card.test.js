import { shallow } from "enzyme";
import React from "react";
import CardList from "./CardList";

it("expect to render Card List component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "John Snow",
      username: "John John",
      email: "john@gmail.com",
    },
  ];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
