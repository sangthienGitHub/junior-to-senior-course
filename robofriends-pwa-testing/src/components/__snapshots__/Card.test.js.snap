// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card List component 1`] = `ShallowWrapper {}`;

exports[`expect to render Card component 1`] = `ShallowWrapper {}`;
